lambdas/src/core/resource/cities/endpoint.rs

```diff
@@ -111,45 +240,112 @@ struct SubmissionParameters {
     pub status: Option<String>,
 }
 
-#[debug_handler]
+#[utoipa::path(
+  get,
+  path = "/cities/submissions/{submission_id}",
+  description = "Get the details of a specific sumission.",
+  tag = TAG,
+  params(
+    ("submission_id" = i32, Path, description = "Submission identifier", example = "1"),
+    ("status" = Option<str>, Query, description = "Filter for the submission status", example = "Pending"),
+  ),
+  responses(
+    (status = OK, description = "Fetches a submission", body = Submissions),
+    ErrorResponses,
+  ))]
 async fn get_cities_submissions(
     Query(submission_params): Query<SubmissionParameters>,
     pagination: Option<Query<PaginationParameters>>,
-) -> Result<Json<Value>, ExecutionError> {
+) -> Result<PageFlow<Submissions>, ExecutionError> {
     let Query(pagination) = pagination.unwrap_or_default();
-    get_cities_submissions_adaptor(
+    let cities_submissions = get_cities_submissions_adaptor(
```

Comment:
same concern as above. Should this be singular or pluaral? `get_cities_submission_adaptor` ?

Reply:
Hmm, looks like there is a mismatch between the endpoint returning a single results, and the endpoint returning multiple ones...

Reply:
Fixed ✅